sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -1,3 +1,7 @@
+# Introduction
+
+Azure Digital Twins is a developer platform for next-generation IoT solutions that lets you create, run, and manage digital representations of your business environment, securely and efficiently in the cloud. With Azure Digital Twins, creating live operational state representations is quick and cost-effective, and digital representations stay current with real-time data from IoT and other data sources. If you are new to Azure Digital Twins and would like to learn more about the platform, please make sure you check out the Azure Digital Twins [official documentation page](https://docs.microsoft.com/azure/digital-twins/overview).
+
```

Comment:
Do you want to include the other deep links Christian provided?

Tutorial 1 — https://docs.microsoft.com/azure/digital-twins/tutorial-code

Tutorial 2 — https://docs.microsoft.com/azure/digital-twins/tutorial-command-line-app

Tutorial 3 — https://docs.microsoft.com/azure/digital-twins/tutorial-end-to-end

Reply:
Sure. I had the impression that the other links will be included elsewhere. But as you mentioned, it makes sense to link these urls to the introduction. I will do a small write up for each link and embed these links in the intro.

Reply:
I think it would be good to list because the topics listed would invite someone to explore. I don't think it needs much more than short blurbs of what they'll find.